### PR TITLE
Prepend language list with recently used languages

### PR DESCRIPTION
**Pull request by scalbatty**
Adresses issue #93 

Rule: when the user explicitly selects a language when posting, when the post is successful, register the selected language as recently used. The recently used language array contains up to 3 distinct items, starting with the most recently used.

Simulator Screen Shot - iPhone 14 Pro - 2023-01-24 at 20 23 49

### CODE REVIEW

File: IceCubesApp/Resources/Localization/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -281,6 +281,7 @@
 "status.editor.drafts.navigation-title" = "Drafts";
 "status.editor.error.upload" = "Error uploading";
 "status.editor.language-select.navigation-title" = "Select Language";
+"status.editor.language-select.recently-used" = "Recently Used";
```
Comment (EvilOne) [inline IceCubesApp/Resources/Localization/en.lproj/Localizable.strings:284]: Please copy this line to the other languages that the app provides. Then, the language updaters will see the English and create a PR to _fix_ it.

Thanks!

Comment (Dimillian) [reply IceCubesApp/Resources/Localization/en.lproj/Localizable.strings:284]: Yep, thanks! This need to be duplicated in every locale files or else the raw value  (key) will show.

Comment (scalbatty) [reply IceCubesApp/Resources/Localization/en.lproj/Localizable.strings:284]: I'm on it

File: Packages/Status/Sources/Status/Editor/Components/StatusEditorAccessoryView.swift [hunk 2]
```diff
@@ -206,22 +225,43 @@ struct StatusEditorAccessoryView: View {
       .font(.scaledCallout)
   }
 
-  private var availableLanguages: [(String, String?, String?)] {
+  private struct Language: Identifiable, Equatable {
```
Comment (Dimillian) [inline Packages/Status/Sources/Status/Editor/Components/StatusEditorAccessoryView.swift:228]: Much better :D

**Comment by Dimillian** [comment]
All good, will merge as soon as the localizations files are updated.

**Pull request merged**